sesame/src/main/scala/org/w3/banana/sesame/SesameTransactor.scala

```diff
@@ -0,0 +1,28 @@
+package org.w3.banana.sesame
+
+import org.openrdf.repository.RepositoryConnection
+import org.w3.banana.{Transactor, SparqlUpdate, RDFStore}
+
+import scala.util.{Failure, Try}
+
+trait SesameTransactor extends Transactor[Sesame,RepositoryConnection] {
+
+  def r[T](conn: RepositoryConnection, body: => T): Try[T] = Try{
+    val result = body
+    conn.close()
```

Comment:
So, I have 3 possible choices:
1) Keep connections open in both r and rw (recommended in bigdata but is different in other sesame databases)
2) Keep r connection open but close rw connection
3) Keep both r and rw connection open.

I think I should look at how default sesame store recommends and do the same as the do (for bigdata we will have another implementation anyway)